src/accelerate/launchers.py

```diff
@@ -142,6 +142,18 @@ def train(*args):
                     "inside your training function. Restart your notebook and make sure no cells initializes an "
                     "`Accelerator`."
                 )
+            # Check for specific libraries known to initialize CUDA that users constantly use
+            already_initialized = are_libraries_initialized("bitsandbytes")
+            if any(already_initialized.values()):
+                err = (
+                    "Could not start distributed process. Libraries known to initialize CUDA upon import have been "
+                    "imported already. Please keep these imports inside your `function` to try and help with this:"
+                )
+                for lib_name, initialized in already_initialized.items():
+                    if initialized:
+                        err += f"\n\t* `{lib_name}`"
+                raise AssertionError(err)
```

Comment:
Not sure about `AssertionError`, I thought they are basically reserved for `assert`s. Maybe use `ImportError` or `RuntimeError`?